### ISSUE TITLE
"Short links" to external services under separate subdomain

### ISSUE THREAD

**Issue by Wituareard** [joepio/pauseai#98]
It could be useful to offer easier to remember redirects to external services, most importantly Discord, under a separate subdomain (to avoid confusion about the redirect). For example, it could be something like `go.pauseai.info/discord` to join the Discord. I'd suggest redirecting the root URL and unknown URLs to a page with a list of the most useful redirects.

I'm not sure how this should be implemented yet. There are externals services like https://short.io/ that offer a limited number of redirects under your own domain for free, but then you are dependent on their goodwill and/or their ability to export data. If the redirects are implemented through the SvelteKit `load` function, I'm not sure how the preload-on-hover feature plays together with that if they are linked somewhere. Maybe we could start with the Netlify `_redirects` file?

**Comment by joepio** [comment]
Good idea! I don't think we need the subdomain, tbh - only makes it longer.

We can have a `/discord` and `/mail` or whatever. Indeed, the `_redirects` file is a great option!

**Comment by Wituareard** [comment]
Hm, I think a subdomain makes sense to avoid the possible confusion of being redirected when clicking on a pauseai.info link.

**Comment by joepio** [comment]
We could internally keep using the direct external links. These are faster anyways.

**Comment by Wituareard** [comment]
Sure, but if someone sends me a normal pauseai.info link, or I see it somewhere else, I'd expect it to take me to a pauseai.info website.

**Comment by Wituareard** [comment]
Okay doesn't seem to work, probably because the function already matches /*

**Comment by joepio** [comment]
My bad, I used the full .org URL instead of just the `/discord`

**Comment by Wituareard** [comment]
It should have worked anyway https://docs.netlify.com/routing/redirects/redirect-options/#domain-level-redirects

**Comment by Wituareard** [comment]
I suspect it's because the edge function matches `/.*` minus `/pagefind/.*` per the generated manifest.json and gets priority before the redirects.

**Comment by joepio** [comment]
I tried to revert their order

**Comment by Wituareard** [comment]
These are the redirects I'd propose. Especially for the more generic ones, I still really think a subdomain would help to avoid confusion.

```js
{
    'shop': 'https://pauseai-shop.fourthwall.com/',
    'store': 'https://pauseai-shop.fourthwall.com/',
    'merchandise': 'https://pauseai-shop.fourthwall.com/',
    'merch': 'https://pauseai-shop.fourthwall.com/',
    'discord': 'https://discord.gg/2XXWXvErfA',
    'x': 'https://x.com/PauseAI',
    'twitter': 'https://twitter.com/PauseAI',
    'linkedin': 'https://www.linkedin.com/company/pauseai',
    'facebook': 'https://www.facebook.com/PauseAI/',
    'tiktok': 'https://www.tiktok.com/@pauseai',
    'instagram': 'https://www.instagram.com/pause_ai/',
    'youtube': 'https://www.youtube.com/@PauseAI',
    'github': 'https://github.com/joepio/pauseai',
    'figma': 'https://www.figma.com/community/file/1233064002969152026',
    'drive': 'https://drive.google.com/drive/folders/1bQ_MZ8giK-Mee4ABkO0BgcFInaXruNpa?usp=sharing',
    'calendar': 'https://calendar.google.com/calendar/u/0?cid=Y19mNWE4YWYyMDZlNjM1ODc2NjVjNmU4MzAzOTgzZmVmYWYzYTBjNjE0NGRiMGFhNDljOTcwZWZhNTEwYTNkODY3QGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20'
}
```

**Comment by Wituareard** [comment]
Using the load function works fine btw, so you'd just have to point it at Netlify, and it could be handled by SvelteKit.
